yarn.lock

```diff
@@ -6300,11 +6300,11 @@ __metadata:
   linkType: hard
 
 "@tailwindcss/container-queries@npm:^0.1.0":
-  version: 0.1.0
-  resolution: "@tailwindcss/container-queries@npm:0.1.0"
+  version: 0.1.1
```

Comment:
I don't see matching changes in `package.json` , perhaps this branch is not yet updated with RC-v1.9 ?

Reply:
Leftover from some package updates I made to the branch (now reverted). Will revert `yarn.lock`